src/outlet/index.ts

```diff
@@ -32,7 +51,7 @@ export class Outlet {
 	urlParams?: URLSearchParams;
 
 	constructor(config: OutletInterface) {
-		this.tokenConfig = config;
+		this.tokenConfig = Object.assign(defaultConfig, config);
```

Comment:
@nicktaras , this assign do single level assign, its not recursive, so be careful here, better assign last-level propertires or assign from root to the end.

Reply:
Thanks @oleggrib - checked in with @micwallace on this, his changes applied are a single level deep from reviewing this together. We will proceed with this to optimise browser performance (ensuring we build update this within flat interface / no nested properties etc).